go-controller/pkg/ovn/namespace.go

```diff
@@ -8,6 +8,29 @@ import (
 	"time"
 )
 
+func (oc *Controller) syncNamespaces(namespaces []interface{}) {
+	expectedNs := make(map[string]bool)
+	for _, nsInterface := range namespaces {
+		ns, ok := nsInterface.(*kapi.Namespace)
+		if !ok {
+			logrus.Errorf("Spurious object in syncNamespaces: %v", nsInterface)
+			continue
+		}
+		expectedNs[ns.Name] = true
+	}
+
+	err := oc.forEachAddressSet(func(addrSetName, asNamespace,
+		asName string) {
```

Comment:
Rename "asName" as "nameSuffix" and "asNamespace" as "namespaceName"

Reply:
Good point, rename asNamespace, asName to namespaceName and nameSuffix.